src/utils/counter.ts

```diff
@@ -0,0 +1,15 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END=
+
+const counter = () => {
+    let _counter = 0;
+
+    return () => {
+        return Number.isSafeInteger(++_counter) ? counter : _counter = 1;
```

Comment:
`counter` is a function in `? counter :`, is that intentional?

Reply:
my bad